core/kamon-core/src/main/resources/META-INF/native-image/io.kamon/kamon-core/reflect-config.json

```diff
@@ -0,0 +1,57 @@
+[
```

Comment:
I think we need add :

```
 {
      "name":"sun.misc.Unsafe",
      "fields":[{"name":"theUnsafe", "allowUnsafeAccess" : true}]
 }
```
In order to avoid :

```
Exception in thread "main" java.lang.NoSuchFieldError: the Unsafe
	at kamon.jsr166.Striped64$1.run(Striped64.java:419)
	at kamon.jsr166.Striped64$1.run(Striped64.java:410)
	at java.security.AccessController.doPrivileged(AccessController.java:123)
	at kamon.jsr166.Striped64.getUnsafe(Striped64.java:410)
	at kamon.jsr166.Striped64.<clinit>(Striped64.java:383)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.invokeClassInitializer(ClassInitializationInfo.java:375)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.initialize(ClassInitializationInfo.java:295)
	at java.lang.Class.ensureInitialized(DynamicHub.java:548)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.initialize(ClassInitializationInfo.java:260)

```
This exception was raised when try to increment a counter `Kamon.counter().increment`. I've tested with this changes and works like a charm ;)

Reply:
Added it. I find it weird that it didn't break with the Quarkus app I was testing :thinking:.. still, I don't think it would hurt anyways.

Reply:
I've tested in a simple main app with `kamon + sbt-native-packager` not a `Quarkus` app ;)